examples/miscellanea/logo.py

```diff
@@ -21,11 +21,12 @@ def main():
     ax.gridlines()
 
     # generate a matplotlib path representing the word "cartopy"
-    fp = FontProperties(family='Bitstream Vera Sans', weight='bold')
+    fp = FontProperties(family='DejaVu Sans', weight='bold')
     logo_path = matplotlib.textpath.TextPath((-175, -35), 'cartopy',
-                                             size=1, prop=fp)
+                                             size=80, prop=fp)
     # scale the letters up to sensible longitude and latitude sizes
-    logo_path._vertices *= np.array([80, 160])
+    transform = matplotlib.transforms.Affine2D().scale(1, 2).translate(0, 35)
+    logo_path = logo_path.transformed(transform)
```

Comment:
Can you pass this as `transform=transform + ccrs.PlateCarree()` in `PathPatch` instead of transforming it here?

Reply:
Sort-of, we can't add the ccrs.PlateCarree() directly, but have to add the MPL version of it. I also changed the image clip-path to just use the patch instead of a separate path as well.

Reply:
I wonder if we should implement `__add__` and `__radd__` to support that.

Reply:
I think that should be a follow-up if someone wants to implement that. The issue will be that the `_as_mpl_transform()` requires an **ax** argument to get the geoaxes currently. There may be a way around that, but it would be a little bit more effort than I was hoping for in this cleanup ;)

Reply:
Ah, good point, maybe that won't work so automatically then.